### PR TITLE
Update Court, Draft, Huddle screens to new UI, also sports card

### PR DESCRIPTION
**Pull request by EphemeralX**
* Update UI to reflect new designs for Court, Draft, Huddle base screens.
* Update sports card UI to match

https://gyazo.com/029736e7153ffa810f886b51a3021677

* Fix RoomView padding w & w/o keyboard on ios and android

### CODE REVIEW

File: civilapp/src/browse/common/SportsCardNew.js [hunk 1]
```diff
@@ -55,60 +53,74 @@ class SportsCardNew extends Component {
...
+                backgroundColor: 'rgba(0,0,0,0.6)'
               }}
-            ></View>
+            >
+              <Image
+                id="OnlineIndicator"
+                style={{
+                  height: size * 0.03,
+                  width: size * 0.03,
+                  // top: '10%',
```
Comment (carteryu) [inline civilapp/src/browse/common/SportsCardNew.js]: rm

File: civilapp/src/browse/BrowseScreen.js [hunk 2]
```diff
@@ -168,17 +164,19 @@ const styles = StyleSheet.create({
...
+    paddingBottom: 2,
+    backgroundColor: '#0D0B5B'
   },
   headerText: {
-    paddingTop: 15,
     color: '#fff',
-    lineHeight: 24,
-    fontSize: 40,
-    fontFamily: UPHEAVAL,
+    lineHeight: 19,
```
Comment (carteryu) [inline civilapp/src/browse/BrowseScreen.js]: we're gonna have this style in literally every file/view right? if so might as well make it an inheritable component

File: civilapp/src/browse/common/SportsCardNew.js [hunk 3]
```diff
@@ -145,65 +157,34 @@ class SportsCardNew extends Component {
                     style={{
                       textAlign: 'center',
                       top: '3.5%',
-                      fontSize: 10,
-                      color: '#78BE20',
+                      fontSize: 8,
+                      lineHeight: 10,
+                      color: '#E76353',//'#78BE20',
```
Comment (carteryu) [inline civilapp/src/browse/common/SportsCardNew.js]: this too

File: civilapp/src/browse/common/SportsCardNew.js [hunk 4]
```diff
@@ -55,60 +53,74 @@ class SportsCardNew extends Component {
...
               style={{
-                flexDirection: 'column',
-                justifyContent: 'flex-start',
-                alignItems: 'flex-start',
-                width: '90%',
-                borderRadius: 5,
-                position: 'absolute',
-                right: '5%',
-                bottom: '50%',
+                flexDirection: 'row',
```
Comment (rauchp) [inline civilapp/src/browse/common/SportsCardNew.js]: none of these styles are dynamic, can we move them to a style sheet?

File: civilapp/src/components/CustomFooter.js [hunk 5]
```diff
@@ -258,7 +259,8 @@ class CustomFooter extends Component {
     const active = this.props.navigation.state.index
 
     return (
-      <LinearGradient colors={gradientColors} style={{ position: 'absolute', bottom: 0, width, height: width / 3, padding: 5, flexDirection: 'row', justifyContent: 'space-around', alignItems: 'center', width, }}>
+      // <LinearGradient colors={gradientColors} style={{ position: 'absolute', bottom: 0, width, height: width / 3, padding: 5, flexDirection: 'row', justifyContent: 'space-around', alignItems: 'center', width, }}>
```
Comment (rauchp) [inline civilapp/src/components/CustomFooter.js]: delete?

**Pull request merged**